<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="containerStyle" stdset="0">
   <string>outer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="containerStyle" stdset="0">
      <string>outer</string>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>535</width>
        <height>541</height>
       </rect>
      </property>
      <property name="containerStyle" stdset="0">
       <string>outer</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="generalSectionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>General</string>
         </property>
         <property name="sectionHeader" stdset="0">
          <bool>true</bool>
         </property>
         <property name="labelStyle" stdset="0">
          <string>title</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="1,20">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="defaultClientUdpPortLabel">
               <property name="text">
                <string>Default Client UDP Port:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="defaultClientUdpPortLineEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="Line" name="line_2">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="hostSectionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Host</string>
         </property>
         <property name="sectionHeader" stdset="0">
          <bool>true</bool>
         </property>
         <property name="labelStyle" stdset="0">
          <string>title</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="clientSectionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Client</string>
         </property>
         <property name="sectionHeader" stdset="0">
          <bool>true</bool>
         </property>
         <property name="labelStyle" stdset="0">
          <string>title</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="1,20">
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="hostStreamResolutionLabel">
               <property name="text">
                <string>Stream Resolution: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="hostStreamResolutionRadio1">
               <property name="text">
                <string>360p</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">hostStreamResolutionButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="hostStreamResolutionRadio2">
               <property name="text">
                <string>720p</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">hostStreamResolutionButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="hostStreamResolutionRadio3">
               <property name="text">
                <string>1080p</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">hostStreamResolutionButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="hostStreamResolutionRadio4">
               <property name="text">
                <string>1440p</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">hostStreamResolutionButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="streamFramerateLabel">
               <property name="text">
                <string>Stream Framerate:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="streamFramerateRadio1">
               <property name="text">
                <string>5 fps</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">hostStreamFramerateButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="streamFramerateRadio2">
               <property name="text">
                <string>12 fps</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">hostStreamFramerateButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="streamFramerateRadio3">
               <property name="text">
                <string>24 fps</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">hostStreamFramerateButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="maxViewCountLabel">
               <property name="text">
                <string>Max Viewer Count: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="maxViewerCountSpinBox">
               <property name="specialValueText">
                <string>3</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="defaultHostUdpPortLabel">
               <property name="text">
                <string>Default Host UDP Port:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="defaultHostUdpPortLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Default Host TCP Port:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="defaultHostTcpPortLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>IP Whitelist:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>IP Blacklist:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QTextEdit" name="whitelist_text"/>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="whitelist_checkbox">
               <property name="text">
                <string>Enabled</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QCheckBox" name="blacklist_checkbox">
               <property name="text">
                <string>Enabled</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2" colspan="2">
              <widget class="QTextEdit" name="blacklist_text"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="1,20">
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="displayNameLabel">
               <property name="text">
                <string>Display Name (max 15 characters):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="displayNameLineEdit"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="darkModeLabel">
               <property name="text">
                <string>Dark Mode:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="darkModeCheckBox">
               <property name="text">
                <string/>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="16" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>80</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="buttonStyle" stdset="0">
        <string>primary</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="hostStreamResolutionButtonGroup"/>
  <buttongroup name="hostStreamFramerateButtonGroup"/>
 </buttongroups>
</ui>
